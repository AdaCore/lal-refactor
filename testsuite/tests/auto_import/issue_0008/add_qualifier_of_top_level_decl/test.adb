with Parent.Child.Great_Child;

procedure Test is
begin
   Great_Child;
end Test;
