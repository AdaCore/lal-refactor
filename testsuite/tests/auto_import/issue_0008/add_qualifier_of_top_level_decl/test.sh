auto_import -P default.gpr --source test.adb --line 6 --column 7
