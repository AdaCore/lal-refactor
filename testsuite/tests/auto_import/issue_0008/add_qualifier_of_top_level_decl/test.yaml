description: >
    This test checks that the tool correctly computes the import and qualifier for a top level
    declaration.

driver: shell_script
