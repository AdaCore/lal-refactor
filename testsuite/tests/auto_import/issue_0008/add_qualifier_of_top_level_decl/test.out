Available Imports:
Import: Parent.Child.Great_Child; Qualifier: Parent.Child

Edits:
test.adb
5:4-5:4 Parent.Child.

